src/components/timeline/TimelineContextMenu.svelte

```diff
@@ -48,6 +49,7 @@
 
   const dispatch = createEventDispatcher<{
     collapseActivityTree: Row;
+    collapseExternalEventTree: Row;
```

Comment:
Unused? Or perhaps collapseActivityTree needs to be refactored to collapseDiscreteTree or something of the sort

Reply:
Addressed in [rename collapseActivityTree to collapseDiscreteTree](https://github.com/NASA-AMMOS/aerie-ui/commit/65fdcb6820c7f4dde2fb6e77e760612225bd424a).